smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -219,28 +221,28 @@ public Task Start()
 
                     if (iotedgeService.Status != ServiceControllerStatus.Running)
                     {
-                        throw new Exception("Can't start up iotedge service within timeout period.");
+                        throw new Exception("Can't start iotedge service within timeout period.");
                     }
+
+                    // Add delay to ensure iotedge service is completely started.
+                    await Task.Delay(new TimeSpan(0, 0, 0, 5));
```

Comment:
Why is this needed? Because iotedged is running but hasn't started the servers yet?

Reply:
iotedged is running; but it is not ready to accept operation request; such as iotedge list.  Therefore 5 sec delay is added.